src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+
+        Colors[] colorValues = Colors.values();
+        Random rand = new Random();
+        int randomIndex = rand.nextInt(colorValues.length - 1);
```

Comment:
Using `length - 1` will never allow the last element of the array to be selected. It should be `rand.nextInt(colorValues.length)` to include all possible indices.